### PR TITLE
Dedupe code opt out

### PR DESCRIPTION
**Pull request by peter-sanderson**
(just last commit)

### CODE REVIEW

File: packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/DeviceAuthenticityOptOutModal.tsx [hunk 1]
```diff
@@ -1,66 +1,31 @@
...
 
-    const handleDeviceAuthenticityOptOut = () => {
-        dispatch(deviceAuthenticityOptOut(true));
-        onCancel();
-    };
-
     return (
-        <StyledModal
-            isCancelable
+        <OptOutModalOptOutModal
```
Comment (komret) [inline packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/DeviceAuthenticityOptOutModal.tsx]: Weird name

File: packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/DeviceAuthenticityOptOutModal.tsx [hunk 2]
```diff
@@ -1,66 +1,31 @@
...
-import { Button, Image, Paragraph } from '@trezor/components';
-import { Translation, CheckItem, Modal } from 'src/components/suite';
+import { Paragraph } from '@trezor/components';
+import { Translation } from 'src/components/suite';
 import { deviceAuthenticityOptOut } from 'src/actions/suite/suiteActions';
 import { useDispatch } from 'src/hooks/suite';
+import { OptOutModalOptOutModal } from './OptOutModal';
 
 const StyledParagraph = styled(Paragraph)`
     text-align: left;
```
Comment (komret) [inline packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/DeviceAuthenticityOptOutModal.tsx]: Not apllied.

File: packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/DeviceAuthenticityOptOutModal.tsx [hunk 3]
```diff
@@ -1,66 +1,31 @@
...
     padding: 16px;
```
Comment (komret) [inline packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/DeviceAuthenticityOptOutModal.tsx]: There should be no additional padding on sides and `spacingsPx` should be used.

File: packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/OptOutModal.tsx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+import styled from 'styled-components';
+import { Button, Image } from '@trezor/components';
+import { Translation, CheckItem, Modal } from 'src/components/suite';
+import { TranslationKey } from '@suite-common/intl-types';
+
+const CheckboxWrapper = styled.div`
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin-top: 32px;
```
Comment (komret) [inline packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/OptOutModal.tsx]: Nothing applied except for margin. Please use `spacingsPx`.

**Review by komret — CHANGES_REQUESTED** [review]

File: packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/DeviceAuthenticityOptOutModal.tsx (hunk 3 above)
Comment (peter-sanderson) [reply packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/DeviceAuthenticityOptOutModal.tsx]: fixed

File: packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/DeviceAuthenticityOptOutModal.tsx (hunk 2 above)
Comment (peter-sanderson) [reply packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/DeviceAuthenticityOptOutModal.tsx]: fixed

File: packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/DeviceAuthenticityOptOutModal.tsx (hunk 1 above)
Comment (peter-sanderson) [reply packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/DeviceAuthenticityOptOutModal.tsx]: fixed

File: packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/OptOutModal.tsx (hunk 4 above)
Comment (peter-sanderson) [reply packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/OptOutModal.tsx]: fixed

File: packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/OptOutModal.tsx [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+const StyledModal = styled(Modal)`
+    width: 600px;
+    ${Modal.Content} {
+        justify-content: center;
+        align-items: center;
+    }
+`;
+
+const WarningImage = styled(Image)`
+    margin: 24px 0;
```
Comment (komret) [inline packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/OptOutModal.tsx]: `spacingsPx` 🙃

File: packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/OptOutModal.tsx [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+`;
+
+const Content = styled.div`
+    margin-left: ${spacingsPx.md};
+    margin-right: ${spacingsPx.md};
```
Comment (komret) [inline packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/OptOutModal.tsx]: `margin: 0 ${spacingsPx.md}` 🙃

**Review by komret — APPROVED** [review]

**Pull request merged**